src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx

```diff
@@ -9,13 +9,9 @@ import { wbsPipe } from '../../../utils/pipes';
 import { useAuth } from '../../../hooks/auth.hooks';
 import ChangesList from '../../../components/ChangesList';
 import DescriptionList from '../../../components/DescriptionList';
```

Comment:
make sure to remove any unused imports like this one

Reply:
Removed all unused imports in "ProjectViewContainer.tsx" and "ProjectDetailTabs.tsx" and checked all other files in the parent folder for unused imports.